dbscripts/xml/upgrade/3.2.0_preupdate_email_templates.xml

```diff
@@ -41,6 +41,13 @@
 		<!-- 4097 = ROLE_ID_ASSISTANT, 768 and 8192 = unused role ids -->
 		<query>UPDATE email_templates_default SET from_role_id=4097 WHERE from_role_id=768 OR from_role_id=8192</query>
 		<!-- 4097 = ROLE_ID_ASSISTANT, 768 and 8192 = unused role ids -->
-		<query>UPDATE email_templates_default SET to_role_id=4097 WHERE to_role_id=768 OR to_role_id=8192</query>
+		<query>
+			UPDATE email_templates_default SET to_role_id=4097 WHERE to_role_id=768 OR to_role_id=8192
```

Comment:
Does this need to move into a newer upgrade?

Reply:
Yes. I changed it to 3.4